content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,143 @@
+</a>
+<br/>
+
+<br/>
+
+
+To see the parking lot data uploaded, we simply navigate to the **Existing** section in the top-right corner of the dashboard once your files have been successfully uploaded.
+
+<br/>
+<a href="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-2.jpg" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-2.jpg" alt="">
+</a>
+<br/>
+
+<br/>
+
+That’s it. Now, My Parking Agent has processed the uploaded data, and let’s set it up for a test to see if it needs further fine-tuning. Click on the **Test AI Agent** button located in the bottom-right corner. Test away and let’s fine-tune the AI agent for optimal performance!.
+
+<br/>
+<a href="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-3.jpg" target="_blank">
+<img width="80%" style="border-radius: 0.4rem;cursor: zoom-in;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-3.jpg" alt="">
+<a/>
+<br/>
+
+## Step  5:  Test Your SeaChat Agent
+
+Before deploying the AI agent we've just created, it’s crucial to train it through testing and fine-tuning. Imagine yourself as the user interacting with the chatbot. You can now start asking any questions to see how your SeaChat agent responds.
+With its updated knowledge base, my AI agent is now ready to be tested. You can start by asking questions to see how your agent responds. Here is what My Parking Agent has to say.
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step5-1.jpg" alt="">
+<br/>
+
+<br/>
+
+To review a message, hover over it and provide positive feedback to your agent with a thumbs up. If the message needs further review, give it a thumbs down.
+Oops, it looks like My Parking Agent needs some fine-tuning. I will give a thumbs down here to provide a feedback.
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step5-02.jpg" alt="">
+<br/>
+
+## Step 6:  SeaChat Agent Fine-Tuning
+
+SeaChat provides a set of sophisticated tools for you to fine-tune your agent to perfection. You can find these amazing features in the **Needs Review** section, accessible via the sidebar menu on the left. Here, you can review the responses that have received your feedback.
+<br/>
+<a href="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step6.jpg" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step6.jpg" alt="">
+</a>
+<br/>
+
+<br/>
+ For My Parking Agent, I just told it to rephrase the way ask a question. Now, it will take in the feedback and rephrase the response accordingly.
+
+<br/>
```

Comment:
I don't really understand the value of this example. Maybe ask about a parking lot that does not exist in the dataset and then add it in the document? 

This example does not seem very critical to agent performance to me

Reply:
This makes a lot more sense. Will do.